### PR TITLE
[new] Added webcamjs@1.0.2

### PR DESCRIPTION
**Pull request by Jeni4**

### CODE REVIEW

**Comment by Jeni4** [comment]
Is it possible someone can help me guess what I do wrong when creating the package.json file __?__ It will seem I am yet to be capable enough to guess it myself. (My first time doing this, so currently a newbie to this) 

Error I get when running _npm test_ on my local machine, but I am currently not able to understand how to read the output...
2015-06-29 00_18_32-c__windows_system32_cmd exe

**Comment by drewfreyling** [comment]
@Jeni4 `filename` is missing in package.json -> https://circleci.com/gh/cdnjs/cdnjs/2567

**Comment by drewfreyling** [comment]
Once the tests pass please squash your commits.

File: ajax/libs/webcamjs/package.json [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+  "maintainers": {
+    "name": "Joseph Huckaby"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git://github.com/jhuckaby/webcamjs.git"
+  },
+  "license": "MIT",
+  "author": "Joseph Huckaby (http://pixlcore.com)"
+}
```
Comment (PeterDaveHello) [inline ajax/libs/webcamjs/package.json]: Need a newline symbol here.

**Comment by Jeni4** [comment]
Commit closed and moved to #4801

**Comment by Jeni4** [comment]
@drewfreyling @PeterDaveHello Thanks for the pointers, did the trick.

**Pull request closed without merging**